TrkReco/src/MergeHelices_module.cc

```diff
@@ -149,7 +149,8 @@ namespace mu2e {
     // Calculate the chi-sq of the helices
     findchisq(evt,h2,chih2xy,chih2zphi);
     // overlapping helices: decide which is best
-    if(nover >= _minnover && nover/float(minh) > _minoverfrac) {
```

Comment:
It seems this change is equivalent to setting _minnover to 0.  Why instead did you remove that test?

Reply:
Yes, I realize now that it is like setting _minnover to 0. The change was made when I noticed events where two tracks are reconstructed for a single particle trajectory. In all such events, the overlapping fraction of hits was > 0.5 but the no. of overlapping hits was just 1 or 2 hits short of _minnover (set to 10). 
[Uploading Screenshot 2023-02-02 at 12.06.26.png…]()

Reply:
Right.  That suggests that cut needs to be re-optimized for your sample, but the cut might still be useful, especially once you include pileup background.  Please leave the test in, and you can adjust the value in fcl .

Reply:
Yes, I understand and can easily adjust the parameter in fcl. I have reverted it back.